trailblazer/server/api.py

```diff
@@ -73,6 +75,19 @@ def get_analyses(analysis_service: AnalysisService = Provide[Container.analysis_
         return jsonify(error=str(error)), HTTPStatus.BAD_REQUEST
 
 
+@blueprint.route("/analyses", methods=["PATCH"])
+@inject
+def patch_analyses(analysis_service: AnalysisService = Provide[Container.analysis_service]):
```

Comment:
A docstring explaining what possible scopes of a patch would be helpful so one can consider the use cases in which this function can be applied.